### DISCUSSION TITLE
开源版本AI智能提示报错

### DISCUSSION THREAD

**Discussion by Jack-GPT** [OtterMind/Chat2DB#1608] [General]
自定义AI我选了智谱，输入了APIkey保存以后，我输入帮我查询一下user的表数据，没反应，然后我用浏览器访问，发现请求报了500，但没有任何错误返回，报错图片：
![1](https://github.com/user-attachments/assets/6158a97e-4ed1-44ca-b5cc-e343569074b1)
![2](https://github.com/user-attachments/assets/30a532bc-485b-4121-9b85-7363e1a38fb3)

**Comment by tmlx1990** [comment]
> 自定义AI我选了智谱，输入了APIkey保存以后，我输入帮我查询一下user的表数据，没反应，然后我用浏览器访问，发现请求报了500，但没有任何错误返回，报错图片： ![1](https://private-user-images.githubusercontent.com/130718199/398295439-6158a97e-4ed1-44ca-b5cc-e343569074b1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwMDY2MzksIm5iZiI6MTczNTAwNjMzOSwicGF0aCI6Ii8xMzA3MTgxOTkvMzk4Mjk1NDM5LTYxNThhOTdlLTRlZDEtNDRjYS1iNWNjLWUzNDM1NjkwNzRiMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQwMjEyMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZGU0NGIxYmQyOGMzZDI4MjM5NWZjNmQ5YzUyMGM3NTJjZTI2MmNjMDczMjE2ZDllMThlODVhNjMwN2QzOTVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.UxNjWAkUlfjZknQSX06V2FsZbwVp4eouNjMtFaJyNHM) ![2](https://private-user-images.githubusercontent.com/130718199/398296093-30a532bc-485b-4121-9b85-7363e1a38fb3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwMDY2MzksIm5iZiI6MTczNTAwNjMzOSwicGF0aCI6Ii8xMzA3MTgxOTkvMzk4Mjk2MDkzLTMwYTUzMmJjLTQ4NWItNDEyMS05Yjg1LTczNjNlMWEzOGZiMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQwMjEyMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00Y2MzNWM5OTU3ZGE4NmJkMjAwNTNmN2RhZTdlYjlhMTliZDI5MTJjMjY0NGQ5YTFiOGE4YWVkNTY5YmJjODY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jaIOa8QaJzhTOChGVc1IM4AObLiIe4zzIQ-7KcU2zq4)

您好，请问您下边两个填了吗？我刚配置了测试是可以的。
<img width="583" alt="image" src="https://github.com/user-attachments/assets/2c716bfc-ba45-41cc-840c-42f23eb3406f" />

<img width="399" alt="image" src="https://github.com/user-attachments/assets/e59a81c1-0cf6-4c10-8a9d-f1504fc0a575" />

**Comment by Jack-GPT** [comment]
好了可以了，感谢，我看不是必填就没填，下面两个输入框里有默认的地址和模型，我也就没管

**Comment by tmlx1990** [comment]
> 好了可以了，感谢，我看不是必填就没填，下面两个输入框里有默认的地址和模型，我也就没管
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>

客气。那个不是默认，只是个提示。
